Automattic/wc-calypso-bridge
# Free trial payment restrictions

Testing instructions

See pdqkbG-2oa-p2 for testing instructions

Additionally, make sure that when the `Set up payments` task is complete, you don't see any other payment gateway suggestions, other than

- PayPal
- Stripe
- WC Pay
- Bacs
- COD
- Cheque

wp-admin/admin.php?page=wc-admin&task=payments

![image](https://user-images.githubusercontent.com/2484390/221116764-701d7941-b88f-45eb-9649-706bf02b61e4.png)


-----
closes #924 


As described in

- peapX7-1rw-p2 ,
- peapX7-1xk-p2

in free trial, payments will be blocked for non-admin users
